CHANGELOG.md

```diff
@@ -0,0 +1,236 @@
+    Plugin](https://plugins.jenkins.io/pipeline-model-definition) works
+    with multibranch
+
+### 2.13
+
+Release date: 2017-03-03
+
+-   [JENKINS-33273](https://issues.jenkins-ci.org/browse/JENKINS-33273)
+    Load `Jenkinsfile` (or any file requested in `loadTrusted`) directly
+    from the SCM rather than doing a checkout. Requires a compatible SCM
+    (currently Git or GitHub).
+-   [JENKINS-40521](https://issues.jenkins-ci.org/browse/JENKINS-40521)
+    Orphaned branch projects are now effectively disabled.
+-   [JENKINS-41146](https://issues.jenkins-ci.org/browse/JENKINS-41146)
+    Help text improvement.
+-   API change: `AbstractWorkflowMultiBranchProjectFactory` implementers
+    may now throw `IOException`.
+
+One aspect of
+[JENKINS-33273](https://issues.jenkins-ci.org/browse/JENKINS-33273) is
+that you will not get an SCM changelog from a lightweight checkout of
+`Jenkinsfile` itself---only if your script actually runs `checkout scm`.
+
+This behavior change can be suppressed in case of emergency using
+`-Dorg.jenkinsci.plugins.workflow.multibranch.SCMBinder.USE_HEAVYWEIGHT_CHECKOUT=true`.
+
+### 2.12
+
+Release date: 2017-02-02
+
+-   [JENKINS-40906](https://issues.jenkins-ci.org/browse/JENKINS-40906) Add
+    missing help text and configuration UI
+
+### 2.10
+
+Release date: 2017-01-16
+
+-   ![(warning)](docs/images/warning.svg)
+      Please read [this Blog
+    Post](https://jenkins.io/blog/2017/01/17/scm-api-2/) before
+    upgrading
+-   [JENKINS-40906](https://issues.jenkins-ci.org/browse/JENKINS-40906) Resolve
+    an SCM from an SCMSource and a list of candidate target branch names
+-   [JENKINS-39355](https://issues.jenkins-ci.org/browse/JENKINS-39355)
+    Use SCM API 2.0.x APIs
+-   [JENKINS-32179](https://issues.jenkins-ci.org/browse/JENKINS-32179) Branch
+    indexing always attempts to create a new project for conflicting
+    branch names from multiple sources
+-   Added subversion based integration tests
+-   If SCMSource.fetch returns null abort the build.
+-   [JENKINS-35698](https://issues.jenkins-ci.org/browse/JENKINS-35698) Initial
+    run of parameterized pipeline build should return properties default
+    value
+-   [JENKINS-38987](https://issues.jenkins-ci.org/browse/JENKINS-38987) SCMHead/SCMSource/SCMNavigator
+    need getPronoun() to assist contextual naming
+-   [JENKINS-38960](https://issues.jenkins-ci.org/browse/JENKINS-38960) Deprecate
+    TopLevelItemDescriptor.getIconFilePathPattern() and
+    TopLevelItemDescriptor.getIconFile(String)
+
+### 2.10-beta-1
+
+Release date: 2016-12-16
+
+-   Updated to use new multibranch-related APIs.
+-   More robust handling of invalid revisions.
+
+### 2.9.2 
+
+Release date: 2016-12-09
+
+-   2.9.1 was corrupt.
+
+### 2.9.1
+
+Release date: 2016-11-09
+
+-   In cases where the SCM source does not recognize a branch and fails
+    to determine its tip revision, fail with a clearer message.
+
+Do not use. Use 2.9.2 instead.
```

Comment:
This should probably be put just under release date in bold. Although at this point it might not matter as its an old release but it makes this bit obvious

Reply:
If you desire you could make a suggestion and I would love to commit it :+1:

Reply:
IMO too small to worry about